Source/WebCore/rendering/ImageQualityController.cpp

```diff
@@ -139,11 +135,19 @@ InterpolationQuality ImageQualityController::chooseInterpolationQuality(Graphics
         }
     }
 
+    const AffineTransform& currentTransform = context.getCTM();
+    bool contextIsScaled = !currentTransform.isIdentityOrTranslationOrFlipped();
```

Comment:
Since you care about the scaling transform only. I think this code can be replaced by:
```
auto scaleFactor = context.scaleFactor();
```

Reply:
Done